etna/articles/models.py

```diff
@@ -226,3 +229,166 @@ def latest_items(self) -> Tuple["ArticlePage"]:
 
     parent_page_types = ["articles.ArticleIndexPage"]
     subpage_types = []
+
+
+class RecordArticlePage(
+    TopicalPageMixin, ContentWarningMixin, TeaserImageMixin, MetadataPageMixin, BasePage
+):
+    parent_page_types = ["collections.ExplorerIndexPage"]
+    subpage_types = []
+
+    standfirst = models.CharField(
+        verbose_name=_("standfirst"), max_length=350, blank=False
+    )
+
+    record = RecordChooserField(verbose_name=_("record"), db_index=True)
+
+    date_text = models.CharField(
+        verbose_name=_("date text"),
+        max_length=100,
+        help_text=_("Date(s) related to the record (max. character length: 100)"),
```

Comment:
Suggestion: We could implement this as a related model to allow querying / better searching?

Reply:
This is deliberately just a text field, as the content can vary. I should probably add a few of these fields to `search_fileds` though.

Reply:
search_fields update in [fe5c58f](https://github.com/nationalarchives/ds-wagtail/pull/604/commits/fe5c58f6e66d3fcfc74df1d525f0db1c56c4b5b3)